ERROR - 2023-03-20 19:35:54 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `mu...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mu...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mu...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mu...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(34): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#11 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-03-20 19:35:54 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `municipios` (`nombre`) VALUES (:nombre:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Municipios.php(34): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 19:47:30 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `mu...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mu...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mu...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mu...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(34): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#11 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-03-20 19:47:30 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `municipios` (`nombre`) VALUES (:nombre:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Municipios.php(34): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 19:47:58 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `mu...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mu...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mu...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mu...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(34): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#11 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-03-20 19:47:58 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `municipios` (`nombre`) VALUES (:nombre:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Municipios.php(34): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 19:48:00 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `mu...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mu...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mu...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mu...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(34): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#11 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-03-20 19:48:00 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `municipios` (`nombre`) VALUES (:nombre:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Municipios.php(34): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 19:52:58 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `mu...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mu...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mu...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mu...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(33): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#11 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-03-20 19:52:58 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `municipios` (`nombre`, `id_dpto`) VALUES (:nombre:, :id_dpto:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Municipios.php(33): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-20 21:11:04 --> syntax error, unexpected token ";", expecting "]"
in APPPATH\Controllers\Municipios.php on line 36.
 1 SYSTEMPATH\Router\Router.php(507): CodeIgniter\Router\AutoRouter->getRoute('municipios')
 2 SYSTEMPATH\Router\Router.php(206): CodeIgniter\Router\Router->autoRoute('municipios')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('municipios')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-20 21:11:05 --> syntax error, unexpected token ";", expecting "]"
in APPPATH\Controllers\Municipios.php on line 36.
 1 SYSTEMPATH\Router\Router.php(507): CodeIgniter\Router\AutoRouter->getRoute('municipios')
 2 SYSTEMPATH\Router\Router.php(206): CodeIgniter\Router\Router->autoRoute('municipios')
 3 SYSTEMPATH\CodeIgniter.php(815): CodeIgniter\Router\Router->handle('municipios')
 4 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 21:21:07 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`)) in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `municip...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `municip...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `municip...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municip...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(39): CodeIgniter\Model->update('56', Array)
#8 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-20 21:21:07 --> Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municipios` SET `nombre` = :nombre:, `id_dpto` = :id_dpto:
WHERE `municipios`.`id` IN :municipios.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Municipios.php(39): CodeIgniter\Model->update('56', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 21:29:52 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`)) in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `municip...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `municip...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `municip...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municip...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(39): CodeIgniter\Model->update('57', Array)
#8 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-20 21:29:52 --> Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municipios` SET `nombre` = :nombre:, `id_dpto` = :id_dpto:
WHERE `municipios`.`id` IN :municipios.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Municipios.php(39): CodeIgniter\Model->update('57', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 21:43:48 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`)) in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `municip...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `municip...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `municip...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municip...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(39): CodeIgniter\Model->update('4', Array)
#8 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-20 21:43:48 --> Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municipios` SET `nombre` = :nombre:, `id_dpto` = :id_dpto:
WHERE `municipios`.`id` IN :municipios.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Municipios.php(39): CodeIgniter\Model->update('4', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 22:46:42 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`)) in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `municip...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `municip...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `municip...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municip...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(48): CodeIgniter\Model->update('58', Array)
#8 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-20 22:46:42 --> Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municipios` SET `nombre` = :nombre:, `id_dpto` = :id_dpto:
WHERE `municipios`.`id` IN :municipios.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Municipios.php(48): CodeIgniter\Model->update('58', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 22:48:14 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`)) in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `municip...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `municip...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `municip...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municip...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(48): CodeIgniter\Model->update('58', Array)
#8 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-20 22:48:14 --> Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municipios` SET `nombre` = :nombre:, `id_dpto` = :id_dpto:
WHERE `municipios`.`id` IN :municipios.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Municipios.php(48): CodeIgniter\Model->update('58', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-20 22:48:33 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`)) in C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\miproyecto1\system\Database\MySQLi\Connection.php(295): mysqli->query('UPDATE `municip...', 0)
#1 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `municip...')
#2 C:\xampp\htdocs\miproyecto1\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `municip...')
#3 C:\xampp\htdocs\miproyecto1\system\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municip...', Array, false)
#4 C:\xampp\htdocs\miproyecto1\system\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\miproyecto1\system\BaseModel.php(943): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\miproyecto1\system\Model.php(756): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\miproyecto1\app\Controllers\Municipios.php(48): CodeIgniter\Model->update('8', Array)
#8 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#9 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#10 C:\xampp\htdocs\miproyecto1\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\miproyecto1\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-20 22:48:33 --> Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2476): CodeIgniter\Database\BaseConnection->query('UPDATE `municipios` SET `nombre` = :nombre:, `id_dpto` = :id_dpto:
WHERE `municipios`.`id` IN :municipios.id:', [...], false)
 2 SYSTEMPATH\Model.php(396): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(943): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Municipios.php(48): CodeIgniter\Model->update('8', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-20 23:28:40 --> Undefined variable $municipios
in APPPATH\Views\municipios\municipios.php on line 39.
 1 APPPATH\Views\municipios\municipios.php(39): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $municipios', 'APPPATH\\Views\\municipios\\municipios.php', 39)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\municipios\\municipios.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('/municipios/municipios', [], true)
 5 APPPATH\Controllers\Municipios.php(24): view('/municipios/municipios', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-20 23:28:41 --> Undefined variable $municipios
in APPPATH\Views\municipios\municipios.php on line 39.
 1 APPPATH\Views\municipios\municipios.php(39): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $municipios', 'APPPATH\\Views\\municipios\\municipios.php', 39)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\municipios\\municipios.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('/municipios/municipios', [], true)
 5 APPPATH\Controllers\Municipios.php(24): view('/municipios/municipios', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-20 23:28:43 --> Undefined variable $municipios
in APPPATH\Views\municipios\municipios.php on line 39.
 1 APPPATH\Views\municipios\municipios.php(39): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $municipios', 'APPPATH\\Views\\municipios\\municipios.php', 39)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\municipios\\municipios.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('/municipios/municipios', [], true)
 5 APPPATH\Controllers\Municipios.php(24): view('/municipios/municipios', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
